src/util/transition/ngbCollapseTransition.ts

```diff
@@ -18,19 +19,22 @@ function measureCollapsingElementHeightPx(element: HTMLElement): string {
     classList.add('show');
   }
 
-  element.style.height = '';
-  const height = element.getBoundingClientRect().height + 'px';
+  element.style[dimension] = '';
+  const dimensionSize = element.getBoundingClientRect()[dimension] + 'px';
 
   if (!hasShownClass) {
     classList.remove('show');
   }
 
-  return height;
+  return dimensionSize;
 }
 
 export const ngbCollapsingTransition: NgbTransitionStartFn<NgbCollapseCtx> =
```

Comment:
You're not cleaning up the `width` in the return of this function, only `height`.